### DISCUSSION TITLE
Suggestion: Add Bromite/Vanadium as System App Browser

### DISCUSSION THREAD

**Discussion by MarkCyberSecDev93** [Divested-Mobile/DivestOS-Build#31] [Ideas] [answered]
Add Bromite/Vanadium as System App Browser and

Use Bromite System Webview/Vanadium System Webview (Recommended by GrapheneOS) As System Webview Replacement. **For Improved Security**

Sorry if some of my suggestions looks a little bit annoying.

Edit: I know Mull is the default browser of DivestOS. What I mean is add an extra system browser like I suggested or replace it (these are my ideas and your choice is yours whether you agree or not).

**Comment by SkewedZeppelin** [comment, marked as answer]
The Mulch WebView used in DivestOS already includes all of the Vanadium patches.

I will not replace it with Bromite, becuase they lag behind releases too much.

https://gitlab.com/divested-mobile/mulch

**Reply by SkewedZeppelin** [reply]
Furthermore:

- You can install the standalone Mulch if you really want.
- Mulch is the only 32-bit builds offering CFI
- The Bromite F-Droid repository is included by default in F-Droid
- Mulch updates get pushed out rapidly to all devices via its F-Droid repo
